lib/spanner/bundle/config_validator.ex

```diff
@@ -51,6 +45,32 @@ defmodule Spanner.Bundle.ConfigValidator do
     validate_commands!(btype, commands)
   end
 
+  # Validate bundle Type
+  defp validate_bundle_type!(json, field) do
```

Comment:
Cool beans. I'll refactor the JsonNavigator code to return :ok or :error and then allow us to handle things accordingly here.